core/opengate_core/opengate_lib/GateSimulationStatisticsActor.cpp

```diff
@@ -103,6 +108,17 @@ void GateSimulationStatisticsActor::PreUserTrackingAction(
   if (fTrackTypesFlag) {
     auto p = track->GetParticleDefinition()->GetParticleName();
     data.fTrackTypes[p]++;
+
+    // For DEBUGGING only - remove
```

Comment:
https://stackoverflow.com/a/561338/1103558
You can check if you're in a debug environment; this will be ignored on release.